public/app/features/api-keys/ApiKeysPage.tsx

```diff
@@ -287,7 +287,14 @@ export class ApiKeysPage extends PureComponent<Props, any> {
                     <td>{key.role}</td>
                     <td>{this.formatDate(key.expiration)}</td>
                     <td>
-                      <DeleteButton onConfirm={() => this.onDeleteApiKey(key)} />
+                      <ConfirmButton
```

Comment:
@torkelo @dprokop I noticed after removing `DeleteButton` we use `ConfirmButton` with the same props in several places. Does it make sense to recreate `DeleteButton` based on new `ConfirmButton`? Maybe do not place it in `@grafana/ui`, but in core grafana as a reusable componen?

Reply:
@alexanderzobnin maybe instead of recreating it, simply replace it with confirm button?

Reply:
@dprokop Maybe. I just don't know what's a better place for this component.

Reply:
I think it can stay as it is imported from grafana/ui

Reply:
@torkelo what do you think?

Reply:
Yea, could be nice with a higher level reusable component